R/module_nested_tabs.R

```diff
@@ -291,9 +297,10 @@ srv_nested_tabs.teal_module <- function(id, datasets, modules, is_module_specifi
     get_datasets_code(datanames, datasets, hashes)
   )
 
+
   do.call(
     teal.data::teal_data,
-    args = c(data, code = list(code), join_keys = list(datasets$get_join_keys()[datanames]))
+    args = c(data, code = list(code), join_keys = list(datasets$get_join_keys()[datanames]), check = check)
```

Comment:
This implementation causes double evaluation - each time when filter is changed.

If the object is verified it shouldn't be re-verified. Instead, `@verified` should be forced to the value of `@verified` slot from the object which has been passed to the application. How to pass this value from `srv_teal` to here?
- not through the modules argument. We have some hacky way to do this https://github.com/insightsengineering/teal/blob/b1bbe12b2e6655db7daac1fc4fc9433a45ee33f5/R/utils.R#L68 
Instead of passing 

```suggestion
    args = c(data, code = list(code), join_keys = list(datasets$get_join_keys()[datanames]))
```

Reply:
Makes sense